MAUI/Cards/Overview.md

```diff
@@ -0,0 +1,25 @@
+---
+layout : post
+title: About MAUI Cards control | Syncfusion
+description: Learn here all about introduction of Syncfusion Essential Studio MAUI Cards control, its elements and more.
+platform : maui
+control : Cards
+documentation : ug
+---
+
+# MAUI Cards Overview
+
+The new Cards control empowers you to generate either a dismissible card or a stack of cards, with only one card being visible initially, requiring a swipe to reveal the subsequent card.
```

Comment:
The new Cards control empowers you to generate either a dismissible card or a stack of cards, with only one card being visible initially, requires a swipe to reveal the subsequent card.